docs/docs/migrate-from/ibm-watson-to-rasa.mdx

```diff
@@ -5,6 +5,54 @@ title: Rasa as open source alternative to IBM Watson - Migration Tips
 description: Open source alternative to IBM Watson for conversational bots and NLP
 ---
 
-There is no support for IBM Watson yet. However, a group of community members is working on a way
-to use <a className="reference external" href="https://developer.ibm.com/tutorials/learn-how-to-export-import-a-watson-assistant-workspace/" target="_blank">exported IBM Watson workspaces</a>
-in Rasa. If you're interested in that, check out our <a className="reference external" href="https://forum.rasa.com/" target="_blank">Community Forum</a>.
+## Step 1 Export your data from Watson
+
+Follow the <a className="reference external" href="https://developer.ibm.com/tutorials/learn-how-to-export-import-a-watson-assistant-workspace/" target="_blank">instructions</a> and export your bot's data.
+
+## Step 2: Create a Rasa Project
+
+To create a Rasa project, run:
+
+```bash
+rasa init
+```
+
+This will create a directory called `data`.
+Remove the files in this directory, and
+move your json file with Watson data into this directory.
+
+```bash
+rm -r data/*
+mv /path/to/file.json data/
+```
+
+## Step 3: Train your NLU model
+
+To train a model using your Watson data, you should convert the data to a format Rasa can understand.:
+
+```bash
+rasa data convert -i data/your-watson-file.json -o data/file.yml --format yaml
```

Comment:
Won't they end up with the json + yaml sitting next to each other under `data`, and Rasa will try to read both? If so probably best to skip the data moving step in step (2) and change this suggestion:
```suggestion
rasa data convert -i path-your-watson-file.json -o data/converted-watson-data.yml --format yaml
```